aten/src/ATen/native/Blas.cpp

```diff
@@ -16,7 +16,7 @@ TORCH_META_FUNC(addmv)(const Tensor &self, const Tensor &mat, const Tensor &vec,
   TORCH_CHECK(mat.size(1) == vec.size(0) && (mat.size(0) == self.numel() || self.numel() == 1),
      "size mismatch, got ", self.size(0), ", ", mat.size(0), "x", mat.size(1), ",", vec.size(0));
   auto names = at::namedinference::propagate_names_for_addmv(mat, vec, self);
-  set_output(0, IntArrayRef(mat.sizes().data(), 1), {}, mat.options(), names);
+  set_output(0, IntArrayRef(mat.sizes().data(), 1), {}, vec.options(), names);
```

Comment:
Why this change?

Reply:
Because `mat` can be now sparse csr tensor while we need strided tensor as output. `vec` is a strided tensor so its options are used to create the output tensor.